app/views/layouts/_page.html.erb

```diff
@@ -0,0 +1,90 @@
+<% content_for :head do %>
+  <meta charset="utf-8">
+  <meta name="theme-color" content="blue">
+  <link rel="manifest" href="manifest.json">
+  <link rel="stylesheet" href="/stylesheets/govuk-frontend.min.css">
+<% end %>
+
+<% content_for :header do %>
+  <%= govuk_header(homepage_url: "https://www.gov.uk", service_name: "Commodity Code Identification Tool Developer Hub", service_url: "#") do |header|
+  header.with_navigation_item(text: "Dashboard", href: "/dashboard", active: false)
+  header.with_navigation_item(text: "Update Profile", href: "#", active: false)
+  header.with_navigation_item(text: "Manage Team", href: "#", active: false)
+  header.with_navigation_item(text: "Sign Out", href: "#", active: false)
+  end %>
+<% end %>
+
+<% content_for :body do %>
+  <div class="govuk-width-container app-width-container">
+    <%= govuk_phase_banner(tag: { text: "Beta" }) do %>This is a new service, your
+      <%= govuk_link_to("feedback (opens in a new tab)", ENV["FEEDBACK_URL"] || 'https://www.trade-tariff.service.gov.uk/feedback', target: "_blank") %>
+      will help us improve it.
+    <% end %>
+    <main class="govuk-main-wrapper app-main-class" id="main-content">
+      <%= yield %>
+    </main>
+  </div>
+<% end %>
+
+<% content_for :footer do %>
+    <div class="govuk-width-container">
```

Comment:
Should this be using govuk_footer? - https://govuk-components.netlify.app/components/footer/